src/DoctrineGui/Form/ClientFieldset.php

```diff
@@ -49,6 +49,20 @@ public function init()
         );
 
         $this->add([
+                'name'          => 'user',

```

Comment:
Either use space padded `=>` to align all elements or don't use space padding at all.  I prefer no space padding.